draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -660,11 +660,6 @@ attribute, the user agent will include the cookie in an HTTP request only if
 the request is transmitted over a secure channel (typically HTTP over Transport
 Layer Security (TLS) {{RFC2818}}).
 
-Although seemingly useful for protecting cookies from active network attackers,
-the Secure attribute protects only the cookie's confidentiality. An active
-network attacker can overwrite Secure cookies from an insecure channel,
-disrupting their integrity (see {{weak-integrity}} for more details).
```

Comment:
Perhaps we could also add a note about `__Secure-` to the "Weak Integrity" section as a partial mitigation? I'll land this and send you a PR to that effect.